lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -406,6 +406,16 @@ def _deploy_nodes(self, environment: Environment, log: Logger) -> None:
             ):
                 self._delete_nodes(environment, log)
 
+            self._log.debug("Capturing libvirtd log from host...")
+            self.host_node.execute(
+                cmd="journalctl -u libvirtd --no-page",
+                sudo=True,
+            )
+            self.host_node.execute(
+                cmd="cat /var/log/libvirt/libvirtd.log",
+                sudo=True,
+            )
+
```

Comment:
can we use journalctl and cat tool?

Reply:
Sure Lili. I will make that change

Reply:
@LiliDeng Done, Please check the latest push

Somehow workflows are not running and being cancelled or skipped. Could you please help me here